std/getopt.d

```diff
@@ -1102,6 +1108,23 @@ unittest
     getopt(args, "fb1", &fb1, "fb2", &fb2, "tb1", &tb1);
     assert(fb1 && fb2 && !tb1);
 
+    // test keepEndOfOptions
+
+    args = (["program.name", "--foo", "nonoption", "--bar", "--", "--baz"]).dup;
```

Comment:
Yeah, it's pretty old. But I think it's far from consistent. You can easily find cases where the `dup` was forgotten. I don't think there was ever a reason to have the `dup`.

In any case, you can remove the `dup` or not, it won't really change anything.